catalogs/catalog-hive/build.gradle.kts

```diff
@@ -118,3 +154,186 @@ tasks {
     dependsOn(copyCatalogConfig, copyCatalogLibs)
   }
 }
+
+var DOCKER_IT_TEST = false
+project.extra["dockerRunning"] = false
+project.extra["macDockerConnector"] = false
+project.extra["isOrbStack"] = false
+
+fun printDockerCheckInfo() {
```

Comment:
You have so many duplications, please improve the code.

Reply:
I tried it, but it doesn't work out at the moment.